src/mantle/framework/console/class-hook-usage-command.php

```diff
@@ -294,7 +294,7 @@ protected function get_cache_file_for_path( string $path ): string {
 	 */
 	protected function set_paths() {
 		if ( ! $this->option( 'search-path' ) ) {
-			$paths = collect( WP_CONTENT_DIR );
+			$paths = collect( defined( 'WP_CONTENT_DIR' ) ? WP_CONTENT_DIR : getcwd() );
```

Comment:
This is not included in the extension as it is not for usage in plugins.

Just create a file - e.g. `tests/phpstan/bootstrap.php`
```php
define( 'WP_CONTENT_DIR', '/' );
```
and add it to `bootStrapFile:` in the configuration.

Reply:
... I know, there is no `contrent_dir()` in core